pkg/provider/securitygroup/securityrule.go

```diff
@@ -141,6 +150,17 @@ func ListDestinationPortRanges(r *armnetwork.SecurityRule) ([]int32, error) {
 	return rv, nil
 }
 
+func SetDestinationPortRanges(r *armnetwork.SecurityRule, ports []int32) {
+	ps := NormalizeDestinationPortRanges(ports)
+	r.Properties.DestinationPortRange = nil
+	r.Properties.DestinationPortRanges = to.SliceOfPtrs(ps...)
+}
+
+func SetAsteriskDestinationPortRanges(r *armnetwork.SecurityRule) {
```

Comment:
Rename to Range? Why do we still need Range, can we use Ranges in all places?

Reply:
for asterisk, only Range works. and for fields like `*prefix` or `*prefixes`, nrp complains if there is only 1 element in `prefixes`. a bit annoying to handle

Reply:
renamed